libcontainer/factory_linux.go

```diff
@@ -256,11 +256,13 @@ func (l *LinuxFactory) StartInitialization() (err error) {
 		if _, ok := i.(*linuxStandardInit); ok {
```

Comment:
Its just writing to stderr and exit(-1) so this change only makes this a single print of the error on stderr and the exit is the same without doing a panic, that is it.